src/schema/resolvers.ts

```diff
@@ -1,9 +1,15 @@
-import { CreateUserInput, UserInput, UserOutput } from './interfaces';
+import { CreateUserInput, LoginOutput, LoginUserInput, UserInput, UserOutput } from './interfaces';
 import { User } from '../entity/User';
 import { AppDataSource } from '../data-source';
 import { createHmac } from 'crypto';
 import { CustomError } from '../format-error';
 
+export function hasher(str: string) {
+  const hash = createHmac('sha256', 'internalizing server behavior');
+  const hashed = hash.update(str).digest('hex');
+  return hashed;
```

Comment:
Poderia retornar direto:

```suggestion
  return hash.update(str).digest('hex');
```